dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/data/TraceWriter.kt

```diff
@@ -0,0 +1,28 @@
+package com.datadog.android.tracing.internal.data
+
+import datadog.opentracing.DDSpan
+import datadog.trace.common.writer.Writer
+
+internal class TraceWriter(
+    val writer: com.datadog.android.core.internal.data.Writer<DDSpan>
+) : Writer {
+
+    override fun start() {
+        // NO - OP
+    }
+
+    override fun write(trace: MutableList<DDSpan>?) {
+        // TODO: RUM-184 Modify the Writer to accept also a list of models
```

Comment:
instead of sending on Span per Runnable we will send all available Spans. More optimised.

Reply:
Oh ok, I didn't understand you were talking about the writer. Because the TODO was in the TraceWriter file I thought you meant the TraceWriter file should accept needed to accept a list of models, hence my confusion.